build-tools-internal/src/main/resources/run.ssl/private-ca.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Do we have to commit these? Can we not leverage security-by-default and have ES auto-generate for us?

Reply:
Having easy access to multiple certificates signed by a known CA (i.e. the one in this PR) and being able to create more sign certificates from the same CA can help with some manual testing efforts. For example, ( and the reason I am making this change) is that we are changing the trust model between multiple clusters. Having these explicit makes manually building and debugging these changes easier. Additionally, the security-by-default only kicks in when no other security features are explicitly configured so it can be challenging to know when/why the auto generation kicks in or doesn't .

Reply:
Fair enough. I assume we generated these with exceedingly long expiration dates so we don't have to worry about regenerating them anytime soon?

Reply:
> with exceedingly long expiration dates 

yup. I calculated my retirement date and added a few years just in case so it won't be my problem when these expire :)  (but hopefully these are only used for this very limited reason)